html/elements/dialog.json

```diff
@@ -0,0 +1,117 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "html": {
+      "elements": {
+        "dialog": {
+          "__compat": {
+            "basic_support": {
+              "support": {
+                "webview_android": {
+                  "version_added": "37.0"
+                },
+                "chrome": {
+                  "version_added": "37"
+                },
+                "chrome_android": {
+                  "version_added": "37.0"
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": false,
+                  "notes": "See <a href=\"https://bugzilla.mozilla.org/show_bug.cgi?id=840640\">bug 840640</a>"
```

Comment:
This is new: we added a rule to use short urls and single quotes here for easier readability. 
Please use `"See <a href='https://bugzil.la/840640'>bug 840640</a>."`
If you rebase, the validator will complain about this, too.

Reply:
Fixed.